WebService/Data/device-simulation-template.json

```diff
@@ -0,0 +1,35 @@
+﻿[
+  {
+    "Id": "1",
+    "Name": "Sample Simple Simulation",
+    "Enabled": false,
+    "IotHubConnectionStrings": [
+      "default"
+    ],
+    "DeviceModels": [
+      {
+        "Id": "truck-01",
+        "Count": 10
+      }
+    ]
+  },
+  {
+    "Id": "2",
+    "Name": "Sample Multiple Devices",
```

Comment:
nit: this is creating simulation with multiple device models (and devices) ? The first one is creating multiple devices as well for single device model